### ISSUE TITLE
🔐 .env for production 

### ISSUE THREAD

**Issue by TimPrd** [TimPrd/Be-Happik#50]
- https://github.com/TimPrd/Be-Happik/blob/409ecb166a59015c96914a6c117c4d62f3aca0c8/%5BBACK%5D/app/controllers/User.js#L199
- https://github.com/TimPrd/Be-Happik/blob/409ecb166a59015c96914a6c117c4d62f3aca0c8/%5BBACK%5D/app/passport.js#L35
- All the mailer.send

**Comment by alexzerah** [comment]
Je n'ai pas bien compris cette issue 🤔 

Est-ce qu'il s'agit de créer une configuration du `.env` pour la production?

**Comment by TimPrd** [comment]
Exact, c'est histoire d'avoir un .env pour la prod sans données sensibles qui traînent ;)
